frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx

```diff
@@ -69,30 +69,33 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
       </DashboardCardHeader>
       <DashboardCardBody>
         <DetailsBody>
-          <DetailItem
-            key="service_name"
-            title="Service Name"
-            value="OpenShift Container Storage"
-            isLoading={false}
-          />
+          <DetailItem key="service_name" title="Service Name" isLoading={false} error={false}>
+            OpenShift Container Storage
+          </DetailItem>
           <DetailItem
             key="cluster_name"
             title="Cluster Name"
-            value={getName(_.get(cephClusterData, 0))}
+            error={!getName(_.get(cephClusterData, 0))}
```

Comment:
can you extract the `getName(_.get(cephClusterData, 0))` so we dont have to call it twice for error and value ?
Same for other values (in other details cards too)

Reply:
ack